package.json

```diff
@@ -70,7 +70,7 @@
     "ultradumbBenchmark": "./lighthouse-core/scripts/benchmark.js",
     "minify-latest-run": "./lighthouse-core/scripts/lantern/minify-trace.js ./latest-run/defaultPass.trace.json ./latest-run/defaultPass.trace.min.json && ./lighthouse-core/scripts/lantern/minify-devtoolslog.js ./latest-run/defaultPass.devtoolslog.json ./latest-run/defaultPass.devtoolslog.min.json",
     "compile-proto": "protoc --python_out=./ ./proto/lighthouse-result.proto && mv ./proto/*_pb2.py ./proto/scripts || (echo \"❌ Install protobuf ≥ 3.7.1 to compile the proto file.\" && false)",
-    "build-proto-roundtrip": "mkdir -p ./.tmp && cd proto/scripts && PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2 python json_roundtrip_via_proto.py",
+    "build-proto-roundtrip": "mkdir -p .tmp && cross-env PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION_VERSION=2 python proto/scripts/json_roundtrip_via_proto.py",
```

Comment:
1) yarn attempts to makes paths OS-agnostic in these commands, but there's a gap wrt `mkdir`. https://github.com/yarnpkg/yarn/issues/7031#issuecomment-523415678 . Can avoid by not using the redundant `./blah` path format to refer to `blah`.
2) cross-env for environment vars
3) removed `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=cpp`. present since the beginning #6183, I think we can do without it. otherwise got this error in windows https://groups.google.com/g/grpc-io/c/jtxyFzYEPEw .

Reply:
historically we've just asked windows devs to use WSL, which lets you run real bash

Reply:
FWIW it's also pretty common to run bash on windows without WSL using gitbash and github actions will even let you select it as your shell of choice instead of powershell if you want to

I still think this is a good change to be making regardless